### PR TITLE
LPS-136245 Add link on Business Home Cover Fragment to Get A Quote Page

### PR DESCRIPTION
**Pull request by MateusCastro**

### CODE REVIEW

File: modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js [hunk 1]
```diff
@@ -64,6 +65,28 @@ continueQuoteButton.onclick = function () {
 		});
 };
 
+getQuoteForm.onsubmit = function (e) {
```
Comment (joseabelenda) [inline modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js:68]: Please, change the parameter name: e -> event

File: modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.css [hunk 2]
```diff
@@ -38,18 +38,18 @@
 	appearance: none;
 	border-color: #7d7e85 !important;
 	flex: 1;
+	font-size: 16px;
```
Comment (joseabelenda) [inline modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.css:41]: Can we use the Style Book typography?

**Review by joseabelenda — CHANGES_REQUESTED** [review]
Good job!
Please, read my comments.
Thx

File: modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.css (hunk 2 above)
Comment (kevenleone) [reply modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.css:41]: Yeah!

File: modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js (hunk 1 above)
Comment (kevenleone) [reply modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js:68]: Alright

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
resent: #104

**Pull request closed without merging**